compass/landice/tests/ismip6_forcing/ocean_basal/process_basal_melt.py

```diff
@@ -0,0 +1,227 @@
+                "-m", mapping_file]
+
+        subprocess.check_call(args)
+
+    def rename_ismip6BasalMelt_to_mali_vars(self, remapped_file_temp,
+                                            output_file):
+        """
+        Rename variables in the remapped ismip6 input data
+        to the ones that MALI uses.
+
+        Parameters
+        ----------
+        remapped_file_temp : str
+            temporary ismip6 data remapped on mali mesh
+        output_file : str
+            remapped ismip6 data renamed on mali mesh
+        """
+
+        # open dataset in 20 years chunk
+        ds = xr.open_dataset(remapped_file_temp, chunks=dict(time=20))
+
+        # build dictionary and rename the ismip6 dimension and variables
+        ismip6_to_mali_dims = dict(
+            ncol="nCells")
+        ds = ds.rename(ismip6_to_mali_dims)
+
+        ismip6_to_mali_vars = dict(
+            deltaT_basin="ismip6shelfMelt_deltaT",
+            gamma0="ismip6shelfMelt_gamma0")
+
+        # drop unnecessary variables on the regridded data on MALI mesh
+        ds = ds.rename(ismip6_to_mali_vars)
+        ds = ds.drop_vars(["lat_vertices", "lon_vertices",
+                           "lat", "lon", "area"])
+
+        # write to a new netCDF file
+        write_netcdf(ds, output_file)
+        ds.close()
+
+    # input files: input uniform melt rate coefficient (gamma0)
+    # and temperature correction per basin
+    _file_basin = "AIS/Ocean_Forcing/imbie2/imbie2_basin_numbers_8km.nc"
+    _files_coeff = {"AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_local_5th_pct_PIGL_gamma_calibration.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_local_5th_percentile.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_local_95th_pct_PIGL_gamma_calibration.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_local_95th_percentile.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_local_median_PIGL_gamma_calibration.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_local_median.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_non_local_5th_pct_PIGL_gamma_calibration.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_non_local_5th_percentile.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_non_local_95th_pct_PIGL_gamma_calibration.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_non_local_95th_percentile.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_non_local_median_PIGL_gamma_calibration.nc",
+                    "AIS/Ocean_Forcing/parameterizations/coeff_gamma0_DeltaT_quadratic_non_local_median.nc"}
```

Comment:
This should be a list, with `[` and `]`.  Currently, it is a `set` because it starts with `{` and ends with `}`.  I believe a set gets sorted somehow to make sure entries are unique.  In any case, that's not a feature we want there.